Storage/SQL/BrowserTable.swift

```diff
@@ -731,7 +770,18 @@ open class BrowserTable: Table {
         }
 
         if from == 0 {
-            // This is likely an upgrade from before Bug 1160399.
+
+            // If we're upgrading from "zero", we may just need to migrate
+            // from using the old `tableList` schema table.
```

Comment:
There are two things in play here:

- Migrating from the `SchemaTable` approach to a version number approach.
- Migrating from _potentially any earlier `SchemaTable` version to the current one.

Worth commenting both of those.